src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -95,6 +95,37 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
     Helpers.parClone(cloneSpecs)
 
   }
+  def snapshotStream(
+                      targetPrefix: String,
```

Comment:
I think we should change the name of this in this func and the existing .snap func. It's too close to target prefix meaning PipelineTable ETLPrefix. I think this should be renamed to something like `snapshotRootPath` or something to make it more clear.